antora.yml

```diff
@@ -55,9 +55,9 @@ asciidoc:
     link-cli-github: https://github.com/che-incubator/chectl
     link-identity-provider-documentation-openshift-3: https://www.keycloak.org/docs/latest/server_admin/#openshift-3
     link-identity-provider-documentation-openshift-4: https://www.keycloak.org/docs/latest/server_admin/#openshift-4
-    link-identity-provider-github: "https://www.keycloak.org/docs/6.0//server_admin/#github"
-    link-identity-provider-oidc: "https://www.keycloak.org/docs/6.0/server_admin/#_oidc"
-    link-identity-provider-saml: "https://www.keycloak.org/docs/6.0/server_admin/#saml-v2-0-identity-providers"
+    link-identity-provider-github: "https://www.keycloak.org/docs/15.0//server_admin/#github"
```

Comment:
@mmorhun It's just the outdated file that fails the links test. I can update this file as part of this PR so that we can have a built preview of the docs for this PR.

If you do find a minute sometime this week to update your branch `che-20337` against upstream `master` at `/che-docs/`, that will also solve this problem.

Reply:
I see, let's keep it then